index.html

```diff
@@ -7,92 +7,226 @@
+            </div>
+            <div class="header_container">
+              <h3 class="paragraph_header">Connection</h3>
+            </div>
+            <div class="paragraph_description flex-j-center">
+              <p class="text">
+                We provide a good internet connection <br>  in our remotest cabins
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo high_way">
+              <img class="pros_pic" src="./highway.png" alt="pic of pros">
+            </div>
+            <div class="header_container flex-j-center">
+              <h3 class="paragraph_header">
+                Accesibility
+              </h3>
+            </div>
+            <div class="paragraph_description">
+              <p class="text">
+                Every Cabin is up in the nature but also <br> easy to get to and to the towns around
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo">
+              <img class="pros_pic" src="./mountains.png" alt="pic of pros">
+            </div>
+            <div class="header_container">
+              <h3 class="paragraph_header">Close to nature</h3>
+            </div>
+            <div class="paragraph_description">
+              <p class="text">Every cabin means different areas to <br> explore!</p>
+            </div>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="highway.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Accesibility</header>
-          <p>Every Cabin is up in the nature but also easy to get to and to the towns around</p>
+    </section>
+    
+    <section class="pink">
+      <div class="flex section_wrapper">
+        <div class="left_side">
+          <div class="some_text">
+            <h2 class="third_page_header">
+              Work from somewhere special
+            </h2>
+            <p class="longer_paragraph">
+              We gather people who love nature and <br> mountains and help them find perfect
+              spots
+              for <br> a few weeks or even
+              months.
+              If your job is fully <br> remote and you daydream of holidays, then why <br> sit at home? Work as you
+              normally
+              do and
+              later <br> let yourself dive into nature. Break the habit.
+            </p>
+          </div>
+          <div class="btn_third_sector">
+            <button class="book_now" type="submit">
+              <span class="book">Book Now</span>
```

Comment:
remove this span, leave plain text in btn

Reply:
Still not done :)

Reply:
fixed